src/store/reducers/storage/types.ts

```diff
@@ -66,6 +82,10 @@ export interface StorageState {
     usageFilter: string[];
     visible: VisibleEntities;
     nodesUptimeFilter: NodesUptimeFilterValues;
+    groupsSortValue?: StorageSortValue;
+    groupsSortOrder?: OrderType;
+    nodesSortValue?: NodesSortValue;
+    nodesSortOrder?: OrderType;
```

Comment:
I split groups and nodes sort params to ensure more strict typing and reduce number of conditions connected with currently selected entity type

Reply:
Moreover I'd better split this into two different reducers in the end.